docs/cinder-csi-plugin/using-cinder-csi-plugin.md

```diff
@@ -88,6 +88,16 @@ In addition to the standard set of klog flags, `cinder-csi-plugin` accepts the f
 
   This will be added as metadata to every Cinder volume created by this plugin.
   </dd>
+
+  <dt>--http-endpoint &lt;HTTP server&gt;</dt>
+  <dd>
+  This argument is optional.
+
+  The identifier of the TCP network address where the HTTP server for diagnostics, including metrics and leader election health check, will listen (example: `:8080`).
```

Comment:
hmm is there leader election health check? I am aware of metrics in /metrics path

Reply:
um... I don't think we do
will update code as well